### ISSUE TITLE
[BUG] Broken search when entering too many results at a time

### ISSUE THREAD

**Issue by olivertzeng** [benbusby/whoogle-search#1095]
**Describe the bug**
Title(Image is much more self-explanatory)

**To Reproduce**
1. Go search for something
2. Do step 1 for about n times
3. ~~profit~~ see error

**Version of Whoogle Search**
- [ ] Latest build from [source] (i.e. GitHub, Docker Hub, pip, etc)
- [x] Version [v0.5.4]
- [ ] Not sure

**Desktop (please complete the following information):**
 - OS: [macOS]
 - Browser [Brave][WaterFox]
 - Version [1.59.124][G6.0.5]

**Additional context**
![截圖 2023-10-31 22 40 29](https://github.com/benbusby/whoogle-search/assets/86348833/5a951bf3-dc09-4005-a3fb-7d10d0b94ea1)

**Comment by benbusby** [comment]
This is a duplicate of an existing issue with Whoogle, but is unlikely to ever get fixed. Since search results are fetched directly from Google, they can use whatever metrics they want to determine if user behavior is "unusual". Some people encounter it more than others, but the only fix seems to be moving Whoogle to be hosted from a new IP. I've heard that Digital Ocean machines have better luck than cheaper VPS services from places like OVH and Hetzner, but can't confirm it personally. Automating CAPTCHA solving is another option, but those services are basically reliant on exploitative labor practices (from what I've been told, haven't researched it extensively myself). 

My only other thought is coming up with some sort of "relay" system where requests that get blocked by a CAPTCHA can be automatically forwarded to a pool of machines that are not getting blocked, but it would be hard to implement without having some users intentionally send continuous spammy searches through the relay to get all other machines blocked as well.
